lib/dotenv/rails.rb

```diff
@@ -43,5 +41,7 @@ def root
     def self.load
       instance.load
     end
+    
```

Comment:
RuboCop has been failed below.

`lib/dotenv/rails.rb:44:1: C: Trailing whitespace detected.`

I think that this warning will disappear with following diff.

```diff
+    
-
```

cf. https://travis-ci.org/bkeepers/dotenv/jobs/172001102

Thanks.